spec/feature/mission_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Missions List', type: :feature do
+  def fill_in_form(mission)
+    fill_in 'Title', with: mission.title
+    fill_in 'Content', with: mission.content
+    fill_in 'Started at', with: mission.started_at
+    fill_in 'Ended at', with: mission.ended_at
+  end
+
+  subject { page }
+  describe 'Showing missions' do
+    context 'when there is no misson' do
+      before { visit missions_path }
+      it_behaves_like 'a missions list page'
+    end
+
+    context 'when there are missions' do
+      let!(:mission) { create(:mission) }
+      before { visit missions_path }
+
+      it_behaves_like 'a missions list page'
+      it_behaves_like 'a page that shows the missions'
+    end
+  end
+
+  describe 'Adding missions' do
+    let(:mission) { build(:mission) }
+
+    before do
+      visit new_mission_path
+      fill_in_form(mission)
+      click_button '建立任務'
+    end
+
+    it_behaves_like 'a missions list page'
+    it_behaves_like 'a page that shows the missions'
+  end
+
+  describe 'Updating missions' do
+    let!(:mission) { create(:mission) }
```

Comment:
此處的 `mission` 好像不需要使用 `#let!` 及 `#create` ？因為：
- 看起來是要給下面的 `fill_in_form(mission)` 填資料用的
- 先 create 的話到時頁面上一定就會有此 mission 的內容，測試就一定會通過

```suggestion
    let(:mission) { build(:mission) }
```